test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -32,6 +32,9 @@
 (deftest ^:integration legacy-routes
   (testing "The legacy web routing service properly handles old routes."
     (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "MOCK.TODO: discuss this one.  Currently all this is testing is that the
```

Comment:
Yep, we've got a couple of threads where we talked about this one.  I think this is addressed now in f476043.